src/client/pages/create.js

```diff
@@ -136,6 +138,21 @@ class Create extends React.Component {
     })
   }
 
+  async copyToClipboard(text) {
+    //Not sure I fully understand binding, should this function be bound to this?
```

Comment:
I found this nice summary: https://gist.github.com/fongandrew/f28245920a41788e084d77877e65f22f

As far as I understand it, in this particular case you don't need to bind as you are using an arrow function in line 246.

Reply:
Thanks, this was really useful